src/main/java/io/vertx/ext/couchdb/admin/impl/CouchdbAdminImpl.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+package io.vertx.ext.couchdb.admin.impl;
+
+import java.util.regex.Pattern;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonArray;
+import io.vertx.ext.couchdb.CouchdbClient;
+import io.vertx.ext.couchdb.admin.CouchdbAdmin;
+import io.vertx.ext.couchdb.database.CouchDbDatabase;
+import io.vertx.ext.couchdb.parameters.DbCreateParams;
+import io.vertx.ext.couchdb.parameters.DbQueryParams;
+
+public class CouchdbAdminImpl implements CouchdbAdmin {
+
+  final CouchdbClient client;
+
+  public CouchdbAdminImpl(CouchdbClient client) {
+    this.client = client;
+  }
+
+  @Override
+  public CouchdbClient getClient() {
+    return this.client;
+  }
+
+  @Override
+  public Future<JsonArray> activeTasks() {
+    String baseUrl = "/_active_tasks";
```

Comment:
Can the baseUrls be moved as constants?

Reply:
It's a single place in the whole app. Not much gained